.azure-pipelines/azure-pipelines-linux-cuda.yml

```diff
@@ -0,0 +1,33 @@
+jobs:
+- job: linux_64_cuda
+  pool:
+    vmImage: ubuntu-16.04
+  strategy:
+    maxParallel: 1
+    matrix:
+      linux_cuda-9_2:
```

Comment:
@jakirkham, I don't understand. This is adding 3 more azure jobs, right?

Reply:
It adds 1 job that sequentially builds each supported CUDA Version.

Reply:
Ah, I guess `jobs` name in azure is slightly different from what is in travis.

Is there a way to skip these builds if there is no cuda needed?

Reply:
You can do a fan out as in https://github.com/conda-forge/_azure_pipelines_test/pull/3

You can do the first job which is the existing one and set the variable `NEED_CUDA` to 0 or 1 depending on whether cuda compiler is used or not.

Then the 3 cuda jobs can run in parallel if `NEED_CUDA` is 1 and if not they are skipped.

Reply:
Yeah I was wondering that same question, but didn't have a good answer at the time. Thanks for investigating.

Fanning out seems reasonable. Guessing this will require an initial parsing of the recipes collected here before building, do you know if there is a good way to hook into the parsing process here to get that information?

Reply:
You can, but if you skip `cuda_compiler_version` == None, then it wouldn't work. So do the easiest way and check for `- {{ compiler('cuda') }}` in the recipe `meta.yaml`. There'll be false positives (if that line is commented), but there won't be false negatives. This can be done in the `build_all.py` where you can just echo that magic line.